### PR TITLE
improve help for -Ddebug

### PR DESCRIPTION
**Pull request by bonzini**
A trivial tweak to docs/help; "debug" is not a very helpful help message.

### CODE REVIEW

**Comment by bonzini** [comment]
Ping for this one line change :) @nirbheek this might also be nice for 0.62.1

File: docs/markdown/Builtin-options.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ machine](#specifying-options-per-machine) section for details.
 | auto_features {enabled, disabled, auto} | auto       | Override value of all 'auto' features                          | no             | no                |
 | backend {ninja, vs,<br>vs2010, vs2012, vs2013, vs2015, vs2017, vs2019, vs2022, xcode} | ninja | Backend to use        | no             | no                |
 | buildtype {plain, debug,<br>debugoptimized, release, minsize, custom} | debug |  Build type to use                    | no             | no                |
-| debug                                | true          | Debug                                                          | no             | no                |
+| debug                                | true          | Include debug information                                      | no             | no                |
```
Comment (nirbheek) [inline docs/markdown/Builtin-options.md:77]: Hm, "Include debug information" is also not very descriptive, right? Maybe it should be "Enable debug symbols and other information"?

Comment (bonzini) [reply docs/markdown/Builtin-options.md]: Done.

**Comment by eli-schwartz** [comment]
This reminds me that once, a long time ago, I wasted an entire afternoon trying to figure out what this option did, because I found it in the `meson setup --help` output and didn't realize it was extended from the coredata options.

Somehow the connection never struck me. I thought it was maybe related to mlog.debug but couldn't even find where the option was defined... the description was far too obtuse to give me any idea of what was going on.

This PR would fix that. :D

**Pull request merged**